index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/bootstrap.min.css">
+    <script src="JS/jquery-3.5.1.min.js"></script>
+    <script src="JS/bootstrap.min.js"></script>
+    <title>Clock Clock Clock</title>
+</head>
+<body>
+    <h1>My First JavaScript Clock</h1>
+    <h3>For all those who can't keep track of time</h3>
+    <canvas id="canvas" class = "active" width="400" height="400" style="background-color:#333"></canvas>

```

Comment:
try and keep your HTML clean, keep a convention.

since you do a lot of <field>=<value>, you should do the same for class. get rid of the spaces. also keep all of these alphabetical.